includes/class-wc-stripe-apple-pay-registration.php

```diff
@@ -412,6 +416,22 @@ public function admin_notices() {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Returns whether Apple Pay is available in the account country.
+	 *
+	 * @since 7.6.0
+	 *
+	 * @return boolean
+	 */
+	private function is_available_in_account_country(): bool {
+		$cached_account_data = WC_Stripe::get_instance()->account->get_cached_account_data();
```

Comment:
I tried injecting `WC_Stripe::get_instance()->account` as a dependency of this class, WC_Stripe_Apple_Pay_Registration, but some unit tests started to fail for WC_REST_Stripe_Settings_Controller for some reason. I debugged it for a bit but preferred not to spend too much time there as this approach is used in other places too. All good with updating it if convenient.

Reply:
I'm still not overly familiar with what the best practices are when working on the Stripe code but this seems totally fine to me 😅